bot/tests/test_issues.py

```diff
@@ -65,3 +68,38 @@ def test_cleaned_paths(log, path, cleaned_path):
         assert log.has(
             "Cleaned issue absolute path", path=path, name="test-task", level="warning"
         )
+
+
+def test_build_hash(mock_revision, mock_hgmo):
```

Comment:
We should add a few more tests (IIRC we had them in the past for local before/after?), at least one to check that the hash doesn't change when the indentation changes.

Reply:
I think it'd also be nice to have a test for each analyzer, so we can clearly see how the hash changes according to the extra data (and maybe we could notice that some extra data should not be part of the hash).

Reply:
Did the first part, filed https://github.com/mozilla/code-review/issues/155 for the second